client/components/web-preview/style.scss

```diff
@@ -22,49 +40,41 @@
 }
 
 .web-preview__content {
-	box-shadow: 0 0 0 1px transparentize( lighten( $gray-dark, 20% ), .5 ),
-		0 1px 2px lighten( $gray-dark, 30% );
+	box-shadow: 0 0 0 1px rgba( $gray, 0.5 );
+	background: $gray-light;
 	border-radius: 4px 4px 0 0;
 	position: absolute;
-		top: 47px;
+		top: 24px;
 		left: 0;
 		right: 0;
 		bottom: 0;
 	margin: 0 auto;
-	transform: scale( 0.96 );
-	transition: 0.1s transform cubic-bezier( 0.25, 0.5, 0.5, 0.9 );
-	background: lighten( $gray, 30% );
+	opacity: 0;
+	transform: translateY( 80vh );
+	transition: transform 0.2s ease-out,
+		opacity 0.1s ease-in-out,
+		max-width 0.2s ease-out;
 
 	.is-computer & {
-		width: 100%;
+		max-width: 1040px;
```

Comment:
I'd prefer to have a fixed with, as large as it could be, and maybe add a fourth icon later with full 100% width. But, take this as a vote of preference. I think that all of the ideas mentioned are overall better than sticking at 1040px.

Trying to get a "good" max number, I checked Analytics for the width in the last months:

* 1366px — 27%
* 1280px — 14%
* 1920px — 13%
* 1440px — 9%
* 1600px — 5%
* 360px — 4%
* 1680px — 2%
* 768px — 2%

So, I think that a good value to set it would be between 1200px and 1400px.